can/interface.py

```diff
@@ -18,7 +18,8 @@
     'nican':            ('can.interfaces.nican', 'NicanBus'),
     'iscan':            ('can.interfaces.iscan', 'IscanBus'),
     'virtual':          ('can.interfaces.virtual', 'VirtualBus'),
-    'neovi':            ('can.interfaces.neovi_api', 'NeoVIBus'),
+    'neovi':            ('can.interfaces.ics_neovi', 'NeoViBus'),
+    'icsneovi':         ('can.interfaces.ics_neovi', 'NeoViBus'),
```

Comment:
Why not keep using "neovi"?

Reply:
Agree, will change to keep the `neovi` interface name.